templates/tests.j2

```diff
@@ -8,7 +8,6 @@ test_format = 1.0
     # Tests to run
     # ------------
 
-{% if data.generator_mode == "tutorial" -%}
     # -------------------------------------------------------------------------------
     # EVERYTHING PAST THIS POINT IS OPTIONAL AND MOST LIKELY UNNECESSARY FOR NEW APPS
     #--------------------------------------------------------------------------------
```

Comment:
Urhgh do we really want to have this big wall of text by default ? Usually this ends up with people keeping those in the final app version and then it looks as if the file is filled with a lot of stuff but it's just all comments (and then comments get outdated because syntax change etc..)

As a middleground we could put a link to https://github.com/YunoHost/package_check?tab=readme-ov-file#teststoml-syntax